data/en/2_0001.json

```diff
@@ -154,7 +154,28 @@
                 },
                 {
                     "id": "number-of-employees-split-block",
+                    "title": "Quarterly Business Survey",
+                    "type": "Question",
                     "questions": [{
+                        "calculated": {
+                            "calculated_type": "sum",
+                            "answer_id": "number-of-employees-total",
+                            "answers_to_calculate": [
+                                "number-of-employees-male-more-30-hours",
+                                "number-of-employees-male-less-30-hours",
+                                "number-of-employees-female-more-30-hours",
+                                "number-of-employees-female-less-30-hours"
+                            ],
+                            "conditions": [
+                                "less than",
+                                "equals"
```

Comment:
Also are we going with less than or equals?

Reply:
its less than or equal for employment